### ISSUE TITLE
[FEATURE REQUEST] Collected/Waitlist banners should match colors set for highlight and/or tag

### ISSUE THREAD

**Issue by grnassar** [tfedor/AugmentedSteam#1165]
### Current Behavior (if any)
Please see images below.
![image](https://user-images.githubusercontent.com/663043/120458188-b0781780-c35c-11eb-924a-2addd9456d86.png)
![image](https://user-images.githubusercontent.com/663043/120458252-be2d9d00-c35c-11eb-8fdf-883fbca18608.png)

### Desired Behavior
"Collected" banner on game page reflecting ITAD collected status should present as the custom color selected by the user for highlight (or, if preferred, tag) for "collected" - here, a drab green color as can be seen in the first image.

### Motivation / Use Case for Changing the Behavior
UI/UX consistency

**Comment by candela97** [comment]
Yeah the color is hard-coded on store pages and I'm not sure why. But it's certainly fixable.
